packages/waku/src/lib/plugins/vite-plugin-rsc-serve.ts

```diff
@@ -59,18 +59,18 @@ export function rscServePlugin(opts: {
           opts.distPublic,
         ),
       };
-      if (opts.serve === 'cloudflare' || opts.serve === 'partykit') {
+      if (opts.serve === 'partykit') {
```

Comment:
@threepointone you might be interested in this PR, as you followed the previous cloudflare impl?